src/mono/mono.proj

```diff
@@ -55,6 +55,8 @@
     <MonoWasmThreads Condition="'$(MonoWasmBuildVariant)' == 'singlethread'">false</MonoWasmThreads>
     <MonoWasmThreads Condition="'$(MonoWasmBuildVariant)' == 'multithread'">true</MonoWasmThreads>
     <MonoEnableAssertMessages Condition="'$(MonoEnableAssertMessages)' == '' and '$(TargetsWasi)' == 'true'">true</MonoEnableAssertMessages>
+    <MonoLLVMMajorVersion>$(runtimelinuxx64MicrosoftNETCoreRuntimeMonoLLVMSdkVersion.Substring(0, $(runtimelinuxx64MicrosoftNETCoreRuntimeMonoLLVMSdkVersion.IndexOf('.'))))</MonoLLVMMajorVersion>
+    <MonoUseLibCxx Condition="'$(MonoLLVMMajorVersion)' >= '16.0'">true</MonoUseLibCxx>
```

Comment:
I'm surprised this works, according to https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2022 you need to escape `>` to `&gt;`

Reply:
It's not clear to me from the docs whether you need to escape `>=` to `&gt;=`, or just `>` to `>=`. The fact that the former works implies it's not required